samples/config/dev/common.yaml

```diff
@@ -0,0 +1,9 @@
+domain: .forgeops.com
+
+image:
+  # repository: gcr.io/engineering-devops
```

Comment:
Might want to switch to images at bintray.io for any public samples so as not to confuse users who find their way here.